code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -2006,3 +2006,136 @@ GLOBAL_LIST_EMPTY(aide_list)
+		return
+	var/obj/item/minihead/B = new()
+	user.put_in_hands(B)
+	to_chat(user, span_notice("You mold a head from the clay!"))
+	COOLDOWN_START(src, next_headcraft, COOLDOWN_HEADCRAFT)
+
+
+
+/obj/item/minihead
+	name = "small clay head"
+	desc = "An earthen homage to its colossal predecessor. The round shape and heft almost scream its desire for flight."
+	icon = 'yogstation/icons/mob/human_parts.dmi'
+	icon_state = "b_golem_head"
+	var/list/headlist = list("abductor_head", "plasmaman_head", "skeleton_head", "cultgolem_head", "fly_head_m", "moth_head_f", "agent_head", "skeleton_head")
+	color = "#774e6d"
+	force = 0
+	throwforce = 0
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF //indestructable bc it needs to exist for a few seconds before dying to not runtime
+	w_class = WEIGHT_CLASS_SMALL
+	var/usedup = FALSE
+	var/smashdam = 25 
+	var/animaldam = 50
+	var/objdam = 25
+
+
+/obj/item/minihead/Initialize(mapload)
+	. = ..()
+	src.icon_state = pick(headlist)
+	
+
+/obj/item/minihead/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	. = ..()
+	if(hit_atom && !usedup)
+		rise(hit_atom, src.icon_state)
+		usedup = TRUE
+
+/obj/item/minihead/after_throw(datum/callback/callback)
+	. = ..()
+	if(!usedup)
+		rise(src.loc, src.icon_state)
+		usedup = TRUE
+
+/obj/item/minihead/attack_self(mob/user)
+	if(!usedup)
+		rise(src, src.icon_state)
+		usedup = TRUE
+
+/obj/item/minihead/attack_hand(mob/user)
+	if(isturf(src.loc) && usedup)
+		return
+	. = ..()
+
+/obj/item/minihead/pickup(mob/user)
+	if(usedup)
+		return FALSE
+	. = ..()
+	
+
+
+/obj/item/minihead/proc/rise(atom/target, icon_state)
+	target.visible_message(span_warning("A stone head grows and floats overhead!"))
+	alpha = 0
+	addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(qdel), src), 2.5 SECONDS)
+	var/obj/item/minihead/shadow = new(get_turf(target))
+	var/obj/item/minihead/realdeal = new(get_turf(target))
+	realdeal.icon_state = icon_state
+	realdeal.usedup = TRUE
+	shadow.icon_state = icon_state
+	shadow.usedup = TRUE
+	shadow.color = "#000000"
+	walk_towards(shadow, target, 0, 0)
+	walk_towards(realdeal, target, 0, 0)
+	animate(realdeal, pixel_y = 60, pixel_x = 0, transform = matrix().Scale(3), time = 1.5 SECONDS)
+	animate(shadow, pixel_y = -30, transform = matrix().Scale(3), time = 1.5 SECONDS)
+	addtimer(CALLBACK(src, PROC_REF(follow), realdeal, shadow), 1.5 SECONDS)
+	return
+
+/obj/item/minihead/proc/follow(var/obj/damocles, var/obj/silhouette)
+	walk(damocles, 0)
+	walk(silhouette, 0)
+	addtimer(CALLBACK(src, PROC_REF(comedown), damocles, silhouette), 0.5 SECONDS)
+	return
+
+/obj/item/minihead/proc/comedown(var/obj/damocles, var/obj/silhouette)
+	var/turf/uhoh = get_turf(damocles)
+	animate(damocles, pixel_y = 0, time = 1 SECONDS, easing = ELASTIC_EASING)
+	animate(silhouette, transform = matrix().Scale(3), time = 0.7 SECONDS)
+	addtimer(CALLBACK(src, PROC_REF(crush), uhoh), 0.04 SECONDS)
```

Comment:
why is this such a small number if the animations last longer than this

Reply:
i dont remember the justification for this oops